src/ops/controlflow.rs

```diff
@@ -209,6 +212,10 @@ impl OpTrait for Case {
     fn tag(&self) -> OpTag {
         <Self as StaticTag>::TAG
     }
+
+    fn signature(&self) -> FunctionType {
```

Comment:
could this return a reference (and clone on use)

Reply:
This is just following the OpTrait interface, but I think they could all just as easily be implemented that way